Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/StringSetting.kt

```diff
@@ -101,7 +101,8 @@ enum class StringSetting(
     companion object {
         private val NOT_RUNTIME_EDITABLE_ARRAY = arrayOf(
             MAIN_CUSTOM_RTC_VALUE,
-            MAIN_GFX_BACKEND
+            MAIN_GFX_BACKEND,
+            GFX_DRIVER_LIB_NAME
```

Comment:
Are you sure this actually works? There's no setting in SettingsFragmentPresenter that's directly mapped to `GFX_DRIVER_LIB_NAME`. Rather, there's an option to open a dialog, and what's inside that dialog is custom and doesn't take `NOT_RUNTIME_EDITABLE_ARRAY` into account.